packages/core/src/pill/Pill.css

```diff
@@ -6,13 +6,13 @@
   background: var(--saltPill-background, var(--pill-background));
   border-radius: var(--salt-palette-corner-weaker, 0);
   border-style: solid;
-  border-width: var(--salt-size-border, 0);
+  border-width: var(--salt-size-fixed-100, 0);
   border-color: var(--pill-borderColor);
   height: calc(var(--salt-size-base) - var(--salt-spacing-100));
   min-height: var(--salt-text-minHeight);
   position: relative;
   gap: var(--salt-spacing-50);
-  padding: 0 calc(var(--salt-spacing-50) - var(--salt-size-border, 0));
+  padding: 0 calc(var(--salt-spacing-50) - var(--salt-size-fixed-100, 0));
```

Comment:
as this is padding should it be spacing-fixed instead of size-fixed?

Reply:
This is an "inside border" calculation so it's referring to the width of the border